R/plumber.R

```diff
@@ -493,7 +493,13 @@ plumber <- R6Class(
 
 #' @rdname plumber
 #' @export
-plumb <- function(file){
+plumb <- function(file, dir){
```

Comment:
As you already observed, we'll need docs for dir. https://github.com/trestletech/plumber/blob/master/R/plumber.R#L25